.circleci/config.yml

```
@@ -268,6 +271,123 @@ jobs:
                 include_job_number_field: false
                 include_project_field: false
                 failure_message: ':red_circle: WordPress Android connected tests failed on \`${CIRCLE_BRANCH}\` branch after merge by ${CIRCLE_USERNAME}. See <https://console.firebase.google.com/u/0/project/api-project-108380595987/testlab/histories/bh.e0c3a59bd9ed670|Firebase console test results> for details.\n\nPlease reach out in #platform9 if you think this failure is not caused by your changes, so we can investigate.'
+  raw-screenshots:
+    docker:
+      - image: circleci/android@sha256:061e2535826cc3fe4c4a440e716bf06c36c80401ee635c339c6803b3e427ebb3
+    environment:
+      JVM_OPTS: -Xmx2048m
+    working_directory: /tmp/workspace
+    steps:
+      - git/shallow-checkout:
+          init-submodules: true
+      - checkout-submodules
+      - bundle-install/bundle-install:
+          cache_key_prefix: v1-raw-screenshots
+      - android/restore-gradle-cache
+      - copy-gradle-properties
+      - update-gradle-memory
+      - restore-gutenberg-bundle-cache
+      - run:
+          name: Ensure assets folder exists
+          command: mkdir -p libs/gutenberg-mobile/gutenberg/packages/react-native-bridge/android/src/main/assets
+      - attach_workspace:
+          at: libs/gutenberg-mobile/gutenberg/packages/react-native-bridge/android/src/main/assets
+      - run: 
+          name: Setup assets
+          command: | 
+            mkdir -p /tmp/workspace/libs/gutenberg-mobile/gutenberg/packages/react-native-bridge/android/build/assets/
+            cp ~/project/libs/gutenberg-mobile/gutenberg/packages/react-native-bridge/android/build/assets/index.android.bundle /tmp/workspace/libs/gutenberg-mobile/gutenberg/packages/react-native-bridge/android/build/assets/index.android.bundle
```

Great find to fix the build issue! 🙇‍♂️ 🙏 

Small nitpick, now that we know this is working and fixing the issue, it could be nice to write it like this in order to make this more resilient (to potential changes of the job's `working_dir` in the future for example) 😉 
```suggestion
            INDEX_BUNDLE_PATH="libs/gutenberg-mobile/gutenberg/packages/react-native-bridge/android/build/assets/index.android.bundle"
            mkdir -p "$(dirname "${CIRCLE_WORKING_DIRECTORY}/${INDEX_BUNDLE_PATH}")"
            cp ~/project/"${INDEX_BUNDLE_PATH}" "${CIRCLE_WORKING_DIRECTORY}/${INDEX_BUNDLE_PATH}"
```

---

Oh, yes! That commit was just to test the solution. Actually, I think that we should fix root issue here, which I think is that there's something wrong about how we attach the workspace. 
I'll check with Gutenberg devs and I hope that we can remove this step.